spec/spec_helper.rb

```diff
@@ -1,7 +1,19 @@
 require "simplecov"
 require "pry"
 SimpleCov.start do
-  track_files "{app,lib}/**/*.rb"
+  add_filter '/db/'
+  add_filter '/spec/'
+  add_group "Models", "app/models"
+  add_group "Controllers", "app/controllers"
+  add_group "Channels", "app/channels"
+  add_group "Decorators", "app/decorators"
+  add_group "Helpers", "app/helpers"
+  add_group "Jobs", "app/jobs"
+  add_group "Importers", "app/lib/importers"
+  add_group "Mailers", "app/mailers"
+  add_group "Policies", "app/policies"
+  add_group "Values", "app/values"
+  add_group "Libraries", "lib"
```

Comment:
If this grouping changed the output in a nice way, let's also add a group for `after_party` tasks, which should have coverage.

Reply:
I replaced the "lib" group with a "/lib/tasks" group.